### player or piece color
uint8 color

### piece enumeration ###
uint8 NoColor =  48 # '0' (ascii zero)
uint8 White   = 119 # 'w'
uint8 Black   =  98 # 'b'
